pbsmetrics/go_metrics.go

```diff
@@ -562,6 +582,14 @@ func (me *Metrics) RecordTimeoutNotice(success bool) {
 	return
 }
 
+func (me *Metrics) RecordTCFReq(version int) {
+	if version >= 0 && version < len(me.TCFReqVersion) {
+		me.TCFReqVersion[version].Mark(1)
+	}
+	me.TCFReqVersion[0].Mark(1)
```

Comment:
Why are we always marking index 0?

Reply:
Oops, should be a return above. Thanks,. `[0]` is the "err" version.